scipy/stats/tests/test_morestats.py

```diff
@@ -996,6 +997,26 @@ def test_mood_alternative(self):
         with pytest.raises(ValueError, match="alternative must be..."):
             stats.mood(x, y, alternative='ekki-ekki')
 
+    @pytest.mark.parametrize("alternative, expected",
+                             [('two-sided', (1.037127561496, 0.299676411857)),
+                              ('less', (1.0371275614961, 0.8501617940715)),
+                              ('greater', (1.037127561496, 0.1498382059285))])
+    def test_mood_alternative_against_R(self, alternative, expected):
+        ## Test againts R mood.test: https://rdrr.io/r/stats/mood.test.html
+        # options(digits=16)
+        # x <- c(111, 107, 100, 99, 102, 106, 109, 108, 104, 99,
+        #             101, 96, 97, 102, 107, 113, 116, 113, 110, 98)
+        # y <- c(107, 108, 106, 98, 105, 103, 110, 105, 104,
+        #             100, 96, 108, 103, 104, 114, 114, 113, 108, 106, 99)
+        # mood.test(x, y, alternative='less')
```

Comment:
I checked this and it doesn't pass as SciPy's implementation of `mood` doesn't correct for ties while R does. I wasn't able to find a way to turn off ties correction in R so, in my suggestion, I used this same data but removed all the ties. Does that make sense to do? Or should we use some other dataset?

Reply:
I see. I didn't understand why you had removed the values. 

I think the best thing to do here it to `xfail` it and create an issue that mood should correct for ties so the xfail can be removed.

Reply:
> I think the best thing to do here it to `xfail` it and create an issue that mood should correct for ties so the xfail can be removed. In the meantime, the `xfail` would serve as a reminder that we have an issue that needs to be addressed. I don't think we should pretend to match R's behavior by hiding the ties.

Makes sense. Let's do that!